.github/workflows/terraform-azdev-agent-pr.yml

```
@@ -0,0 +1,47 @@
+        env:
+          SOURCE_DATE_EPOCH: ${{ env.TIMESTAMP }}
+      - name: Scan with trivy
+        uses: aquasecurity/trivy-action@0.29.0
+        with:
+          image-ref: terraform-azure-devops-agent:test
+          format: table
+          exit-code: 1
+          ignore-unfixed: true
+          trivyignores: ./infrastructure/images/terraform-azure-devops-agent/.trivyignore
+          vuln-type: os,library
+          severity: HIGH,CRITICAL
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)